packages/byods/src/token-manager/index.ts

```diff
@@ -137,25 +147,29 @@ export default class TokenManager {
     headers: Record<string, string> = {}
   ): Promise<void> {
     try {
-      const response = await httpUtils.post<TokenResponse>(
-        `${this.baseUrl}/applications/${this.serviceAppId}/token`,
-        {
-          headers: {
-            Authorization: `Bearer ${personalAccessToken}`,
-            'Content-Type': 'application/json',
-            ...headers,
-          },
-          body: JSON.stringify({
-            targetOrgId: orgId,
-            clientId: this.clientId,
-            clientSecret: this.clientSecret,
-          }),
-        }
-      );
+      const response = await httpUtils.post<TokenResponse>(`${this.baseUrl}/access_token`, {
```

Comment:
Prettier formatted it like this... not sure why

Reply:
@adhmenon I had made initial changes causing Syntax error so based on the format suggest by the ChatGPT,I had implemented it